packages/plugin-phone/src/call.js

```diff
@@ -389,7 +389,9 @@ const Call = SparkPlugin.extend({
       this.stopListening(this.spark.mercury);
       this.off();
       URL.revokeObjectURL(this.localMediaStreamUrl);
+      this.localMediaStreamUrl = undefined;
```

Comment:
This is the only "fix" I put in for the plugin to match the tests. We will do more in another PR.